apps/testapp/src/components/RpcMethods/method/walletTxMethods.ts

```diff
@@ -0,0 +1,44 @@
+import { parseMessage } from '../shortcut/ShortcutType';
+import { RpcRequestInput } from './RpcRequestInput';
+
+const walletGetCapabilities: RpcRequestInput = {
+  method: 'wallet_getCapabilities',
+  params: [],
+};
+
+const walletSendTransaction: RpcRequestInput = {
+  method: 'wallet_sendTransaction',
+  params: [
+    { key: 'chainId', required: true },
+    { key: 'sender', required: true },
+    { key: 'calls', required: true },
+    { key: 'capabilities', required: true },
```

Comment:
capabilities are optional. also missing here are `gas` and `version`. `gas` is optional and `version` is required

Reply:
thanks, will update

Reply:
addressed in [6178c33](https://github.com/coinbase/coinbase-wallet-sdk/pull/1138/commits/6178c33e20166ef8e819447e8ed74dbd1299dfb7)